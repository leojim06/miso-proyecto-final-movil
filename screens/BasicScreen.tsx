import React from "react";
import { Text, View } from "react-native";

export default function BasicScreen(){
    return(
        <View>
            <Text>Hola mundo</Text>
        </View>
    )
}