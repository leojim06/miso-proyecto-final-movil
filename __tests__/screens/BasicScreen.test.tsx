import React from 'react';
import renderer from 'react-test-renderer';

import BasicScreen from '../../screens/BasicScreen';

describe('<BasicScreen />', () => {
    it('has 1 child', () => {
        const tree: any = renderer.create(<BasicScreen />).toJSON();
        expect(tree?.children?.length).toBe(1);
    });
    it('App renders without crashing', () => {
        const rendered = renderer.create(<BasicScreen />).toJSON();
        expect(rendered).toBeTruthy();
    });
    it('App test against snapshot', () => {
        const tree = renderer.create(<BasicScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
