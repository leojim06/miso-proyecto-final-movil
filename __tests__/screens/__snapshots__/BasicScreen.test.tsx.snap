// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasicScreen /> App test against snapshot 1`] = `
<View>
  <Text>
    Hola mundo
  </Text>
</View>
`;
